contrib/mingw-w64-headers/template.py

```diff
@@ -0,0 +1,70 @@
+pkgname = "mingw-w64-headers"
+pkgver = "11.0.1"
+pkgrel = 0
+build_wrksrc = "mingw-w64-headers"
+build_style = "gnu_configure"
+hostmakedepends = ["autoconf", "automake", "libtool"]
+pkgdesc = "Header files for Windows"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "ZPL-2.1"
+url = "https://www.mingw-w64.org"
+source = f"$(SOURCEFORGE_SITE)/mingw-w64/mingw-w64-v{pkgver}.tar.bz2"
+sha256 = "3f66bce069ee8bed7439a1a13da7cb91a5e67ea6170f21317ac7f5794625ee10"
+options = ["empty"]
+
+_targets = ["x86_64", "i686", "aarch64", "armv7"]
+
+for _an in _targets:
+    self.depends += [f"mingw-w64-headers-{_an}={pkgver}-r{pkgrel}"]
```

Comment:
self does not belong here

Reply:
you still need to pre-declare an empty depends (can't append to an undeclared variable)